ebs-backup.py

```diff
@@ -78,7 +80,18 @@ def backup_instance(instance):
             'DeleteOn': delete_date_fmt
         }
         tag_list = list(map(lambda kv: {'Key': kv[0], 'Value': kv[1]}, list(tags.items())))
-        client.create_tags(Resources=snapshot_ids, Tags=tag_list)
+        client_error = None
+        for i in range(6):
```

Comment:
should this `i` be an `e` in my vscode pylint reports it as `unused variable i`

Reply:
No, it's just a standard for loop but I don't need the number. Sure there would be workarounds to ditch that variable but it's not really worth caring about in this instance.